### DISCUSSION TITLE
How to dowload a file with wildcard

### DISCUSSION THREAD

**Discussion by pheiduck** [curl/curl#12493] [Q&A] [answered]
for a https request?
https://github.com/user/repo/releases/download/*/file-*.zip
* = version number 1.2.3

**Comment by dfandrich** [comment, marked as answer]
This isn't possible with HTTP. FTP provides a standardized way of listing files
in a directory so curl can glob those, but with HTTP, there's no generalized
equivalent. You can still use curl's [] and {} globbing functions but you need
to be specific about the URLs you want. Unlike wget, curl doesn't parse HTML to
get lists of possible URLs to download.
